ui/apps/platform/cypress/constants/VulnManagementPage.js

```diff
@@ -32,6 +32,21 @@ export const url = {
     },
 };
 
+/*
+ * Headings on entities pages: uppercase style hides the inconsistencies.
+ * The keys correspond to url list object above.
+ */
+export const headingPlural = {
+    clusters: 'clusters',
+    components: 'components',
+    cves: 'CVES',
+    deployments: 'deployments',
+    images: 'images',
+    namespaces: 'namespaces',
+    nodes: 'nodes',
+    policies: 'policies',
+};
```

Comment:
That is an alert observation. Assertions like this in new helper functions have 2 purposes:
1. Make the assertion automatic without need to include in any particular test.
2. Use the assertion as another opportunity to wait, in case there is any loophole in waiting on requests, before tests access DOM elements. Especially helpful for older tests where cost and benefit is not enough to rewrite jQuery-style method chains as atomic selectors (which take full advantage or cypress retry).